migrations/2022-01-26-184200_index_action_receipt_actions/up.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX CONCURRENTLY action_receipt_actions_receiver_and_timestamp_idx
```

Comment:
It seems we learned that diesel migration tool cannot really use `CONCURRENTLY` in the migrations since it does it wraps the migrations in transactions